admin-jobs/app/models/BreakingNews.scala

```diff
@@ -0,0 +1,61 @@
+package models
+
+import java.util.UUID
+import play.api.libs.json._
+import play.api.libs.functional.syntax._
+
+case class BreakingNewsCollection(name: String, href: String, id: UUID, alerts: Set[NewsAlertNotification])
+
+case class BreakingNews(alerts: Set[NewsAlertNotification]) {
+
+  val collections = List(
+    BreakingNewsCollection("UK alerts", "uk", UUID.fromString("5dff149e-3cc8-4b23-ba7d-c7596d8e39e5"), alerts.filter(_.isOfType(NewsAlertTypes.Uk))),
```

Comment:
I don't think the UUID for each collection is used anywhere. I kept it anyway. Let me know if this is unecessary